holder.yml

```diff
@@ -56,60 +56,108 @@ paths:
           description: invalid input!
         "500":
           description: error!
-  /presentations/available:
+  /exchanges/{exchange-id}:
     post:
-      summary: Notifies a holder of an available presentation.
-      operationId: notifyPresentationAvailable
+      summary: Initiates an exchange of information.
+      operationId: initiateExchange
       description:
-        This API is the first part of a holder to holder credential exchange over http.
-        Notifies this server-holder that the client-holder (caller) is prepared to deliver a presentation compliant with the provided Verifiable Presentation Request query. This is the first part of a presentation exchange over this HTTP API. This server-holder MUST return the required 'domain' and 'challenge' parameters for the Presentation. This server-holder MAY reply with an alternate Verifiable Presentation Request query if it requires something different than the one proposed by the client-holder. The client-holder MUST return a Presentation in accordance with the response parameters.
+        A client can use this endpoint to initiate an exchange of a particular
+        type. The client can include HTTP POST information related to the
+        details of exchange it would like to initiate. If the server understands
+        the request, it returns a Verifiable Presentation Request. A request
+        that the server cannot understand results in an error.
+      parameters:
+        - in: path
+          name: exchange-id
+          required: true
+          schema:
+            type: string
+            minimum: 3
+            pattern: "[a-z0-9\\-]{3,}"
```

Comment:
This is a very western-centric regex. I expect people to object to the fact we're not allowing non-western characters in here (in time).

Reply:
Why would we encourage so much diversity in characters for an id that all parties need to agree too?

See https://github.com/json-schema-org/json-schema-spec/issues/542

I think we should consider if this `id` also shows up inside VPs and VCs and if so, is in a proper IRI or not.

I prefer for it to be `urn:uuid:`.

Reply:
> Why would we encourage so much diversity in characters for an id that all parties need to agree too?

Not all parties need to agree to this identifier. It is (or can be) issuer-specific. 

> I prefer for it to be urn:uuid:

I don't understand this desire, so we should discuss on the next VC API call. In any case, my suggestion is that we deal w/ the range of exchange-id later.